README.md

```diff
@@ -1,115 +1,105 @@
 
 # Thumbnailer
 
-Thumbnailer takes timestamped images as an input and produces an animation in WebP format that best fits under a given size constraint.
+Thumbnailer takes timestamped images as input and produces a WebP animation that fits a given size constraint.
 
-## Prerequisites
+## Building
 
-- [Bazel Build System](https://docs.bazel.build/versions/master/bazel-overview.html)
-- [Clang](https://clang.llvm.org/)
+#### Prerequisites
 
-On Linux, the following would install all the dependencies you need for building the API:
-```
-sudo apt-get install bazel clang unzip zip
-```
+- [Bazel](https://docs.bazel.build/versions/master/install.html)
+- [Clang](https://clang.llvm.org/)
 
-## Building
+#### Build command
 
-In order to build the Thumbnailer binaries, you can use the following Linux command line:
 ```
-bazel build "//src:thumbnailer" && bazel build "//src/utils:thumbnailer_compare"
+bazel build src:thumbnailer && bazel build src/utils:thumbnailer_compare
 ```
 
 ## Usage
 
-### Thumbnail Generating Tool
+### Thumbnailer
 
-This tool reads a sequence of images and timestamps from a text file and produces an animation in WebP format.
+This tool reads a sequence of images and timestamps from a text file and produces an animation in WebP format. By default, `thumbnailer` uses lossy compression and imposes the same quality factor to all frames.
 
 #### Usage
 
 ```
 ./bazel-bin/src/thumbnailer [options] frame_list.txt -o=output.webp
 ```
 
-By default, the lossy compression method imposing the same quality to all frames will be used to produce animation. 
-
-#### Example:
-
-Command line to generate animation using various options:
+or 
 
 ```
-./bazel-bin/src/thumbnailer -loop_count 5 -verbose -slope_optim frames.txt -o anim.webp
+bazel run src:thumbnailer [options] frame_list.txt -o=output.webp
```

Comment:
You could trim here by keeping only one of the two usage command lines

Reply:
The `bazel run` is easier to use, but it also runs `build` and rebuilds the binary.